src/Client/widgets/FilterBlock/View.fs

```diff
@@ -0,0 +1,24 @@
+module FilterBlock.View
+
+open Fable.Helpers.React
+open Fable.Helpers.React.Props
+open Fable.FontAwesome
+
+open Types
+
+let private filterBlockView isHidden dispatch children =
+    div [ ClassName ("filter-block " + if isHidden then "children-hidden" else "") ]
```

Comment:
The "children-hidden" class is added in case toggle is used, on desktop this class has no style difference with no class.